pgrx/src/datum/mod.rs

```diff
@@ -149,3 +149,23 @@ impl<'src> Datum<'src> {
 /// A tagging trait to indicate a user type is also meant to be used by Postgres
 /// Implemented automatically by `#[derive(PostgresType)]`
 pub trait PostgresType {}
+
+/// Creates a [`Vec<pg_sys::Oid>`] containing identifiers of the provded types.
+///
+/// # Examples
+///
+/// ```
+/// use pgrx::{oids_of, datum::IntoDatum};
+/// let v = oids_of![i32, f64];
+/// assert_eq!(v[0], i32::type_oid().into());
+/// assert_eq!(v[1], f64::type_oid().into());
+/// ```
```

Comment:
mostly for the last line, because I'm silly
```suggestion
///
/// // the usual conversions or coercions are available
/// let oid_vec: Vec<pg_sys::Oid> = oids_of![i8, i16].to_vec();
/// let no_oid: &[pg_sys::Oid] = &oids_of![];
/// assert_eq!(no_oid.len(), 0);
/// ```
```

Reply:
Not sure about it, so applied it partly. I made it mostly to demonstrate that the resulting type of `oids_of!` is known even for an empty set of types.